synapseclient/__main__.py

```diff
@@ -550,20 +561,30 @@ def build_parser():
             help='Synapse ID of project or folder where file/folder will be moved ')
     parser_mv.set_defaults(func=move)
 
-    # parser_cp = subparsers.add_parser('cp',
-    #         help='Copies a file in Synapse')
-    # parser_cp.add_argument('--id', metavar='syn123', type=str, required=True,
-    #         help='Id of entity in Synapse to be copied.')
-    # parser_cp.add_argument('--parentid', '--parentId', '-parentid', '-parentId', metavar='syn123', type=str, required=True, dest='parentid',
-    #         help='Synapse ID of project or folder where file will be moved ')
-    # parser_cp.add_argument('-v', '--version', metavar='VERSION', type=int, default=None,
-    #         help='Synapse version number of entity to retrieve. Defaults to most recent version.')
-    # parser_cp.add_argument('--setProvenance', metavar='MODE', type=str, default='traceback',
-    #         help=('Has three values to set the provenance of the copied entity-'
-    #                     'traceback: Sets to the source entity'
-    #                     'existing: Sets to source entity\'s original provenance (if it exists)'
-    #                     'None/none: No provenance is set'))
-    # parser_cp.set_defaults(func=copy)
+    parser_cp = subparsers.add_parser('cp',
+            help='Copies a file in Synapse')
```

Comment:
Copies synapse content such as files, folders and project by recursivey copied all sub-content